apmpackage/apm/data_stream/app_metrics/manifest.yml

```diff
@@ -23,6 +23,10 @@ elasticsearch:
                 - sum
                 - value_count
               default_metric: value_count
+        - doubles:
+            match_mapping_type: float
```

Comment:
```suggestion
```

We should always explicitly specify the dynamic template by name, so it shouldn't be necessary to match on type.